content/background.html

```diff
@@ -1,19 +1,18 @@
-<h1 id="background"> Background </h1>
+<h1 id="background"> Demographic & Background</h1>
 <h3> Gender </h3>
 <p class="half">
   <span class="big">
-    Women make up 17% of students, which is approximately the same proportion as
-    the number of women in tech.
+  Women make up 21% of students, which is less than the percent of women in tech (25%).
   </span>
-  Although we are still working towards increasing the number of minority gender
-  groups, we can see a positive increase within the recent SE classes
+  The class saw a 3% increase in women representation from 1A to 4B, in first year, 
+  18% of the class were female.
   <br /> <br />
-  There are also many on-campus communities that supports the success of women in
-  computer science at UWaterloo. For example,
-  <a href="http://wics.uwaterloo.ca/about/">Women in Computer Science</a>, and
+  There are many on-campus communities that supports the success of women in
+  computer science and engineering at UWaterloo. For example,
+  <a href="https://sesoc.uwaterloo.ca/wise">Women in Software (WiSE)</a>,
+  <a href="http://wics.uwaterloo.ca/about/">Women in Computer Science (WiCS)</a>,
+  <a href="https://uwaterloo.ca/women-in-engineering/">Women in Engineering (WiE)</a>, and
   <a href="https://www.meetup.com/en-AU/Women-Who-Code-Waterloo/"> Women Who Code Waterloo </a>.
-  <br /><br />
-  One student preferred not to disclose.
 </p>
 <div class="half second">
   <span class="graph-title">Gender proportion in class</span>
```

Comment:
The entry for "non-binary" overlaps with "prefer not to disclose" on my display (13" 2015 mbp pro). 
<img width="330" alt="Screen Shot 2021-05-26 at 1 14 52 PM" src="https://user-images.githubusercontent.com/10256087/119703267-5d660800-be24-11eb-95f1-d55fe52162d8.png">

Reply:
Yes, I am aware of this. I was planning to fix this in a later version, cause I think the pi graph still makes sense to use, but I guess a horizontal bar graph does the same job too

Reply:
It's alright, we can mark a TODO for this. Pie does make the most sense